include/RAJA/config.hxx.in

```diff
@@ -154,102 +154,106 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //
 //     RAJA_INLINE - macro to enforce method inlining
 //
-//     RAJA_ALIGN_DATA(<variable>) - macro to express alignment of data, 
-//                              loop bounds, etc.
+//     RAJA_SIMD - macro to apply compiler-specific pragma that encourages
+//                 compiler to SIMD vectorize loop even if it may not
+//                 normally
+//
+//     RAJA_SIMDFORCE - macro to apply compiler-specific pragma that forces
```

Comment:
Sure, but maybe those policy names confuse things. I think that: scalar (novec), sequential (compiler), simd (force) could be better.

This would make sequential keep its current behavior, have simd work as it should, and have scalar ensure things aren't vectorized.